tests/numpy/test_stack.py

```diff
@@ -0,0 +1,84 @@
+# coding=utf-8
+# Copyright (C) 2020 NumS Development Team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import numpy as np
+
+
+# pylint: disable=import-outside-toplevel
+
+
+def test_atleast_1d(nps_app_inst):
+    import nums.numpy as nps
+    assert nps_app_inst is not None
+
+    x = 1.0
+    assert np.allclose(nps.atleast_1d(x).get(), np.atleast_1d(x))
+
+    x_np = np.arange(9).reshape(3, 3)
+    x_nps = nps.arange(9).reshape(3, 3)
+    assert np.allclose(nps.atleast_1d(x_np).get(), np.atleast_1d(x_np))
+    assert np.allclose(nps.atleast_1d(x_nps).get(), np.atleast_1d(x_np))
+    assert nps.atleast_1d(x_nps) is x_nps
```

Comment:
Can you test that this `is` check is also true for numpy?

Reply:
Yep.